doc/installation_instructions/basic.rst

```diff
@@ -0,0 +1,300 @@
+******************
+Basic Installation
+******************
+
+The following will show how to install both SmartSim and SmartRedis.
+
+.. note::
```

Comment:
I do not think note: is giving the correct format on the page

Reply:
Nice catch!